app/services/bulk_upload/lettings/year2024/csv_parser.rb

```diff
@@ -0,0 +1,113 @@
+require "csv"
+
+class BulkUpload::Lettings::Year2024::CsvParser
+  include CollectionTimeHelper
+
+  FIELDS = 130
+  MAX_COLUMNS = 142
```

Comment:
do we not want to reduce this by 4 as there are 4 fewer fields?

Reply:
So EL is 142, EA is 131 so will update it to that. Thank you!